fetch.bs

```diff
@@ -6653,6 +6689,153 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
 </ol>
 </div>
 
+<h3 id=deferred-fetching>Deferred fetching</h3>
+
+<p>Deferred fetches allow callers to request that a fetch is invoked at the latest possible moment,
+i.e. when a <a>fetch group</a> is <a for="fetch group">terminated</a>, or after a timeout.
+
+<div algorithm="request-a-deferred-fetch">
```

Comment:
"request a deferred fetch" (no hyphens)

But also is this needed? It seems there's only a single `<dfn>` here so we don't need to supply a name.

Reply:
Removed